client/modules/Agency/components/AgencyListItem/AgencyListItem.css

```diff
@@ -1,18 +1,20 @@
-.single-category {
-  margin: 20px 0;
-  padding: 15px;
-  border-radius: 2px;
+.single-agency {
+  margin: 10px 0;
+  border-radius: 1px;
   border: solid grey;
+  text-align: center;
+  background: lightgrey;
+  width: inherit;
 }
 
-.category-title{
+.agency-title{
   font-size: 28px;
   margin-bottom: 16px;
```

Comment:
This makes the text not vertically centered, not critical but caught my eye.

Reply:
Still gonna approve

Reply:
Done